excel2sbol/excel2sbol/comp_column_functions.py

```diff
@@ -415,7 +416,7 @@ def objectType(self):
         pass
 
     def types(self):
-        # used to decide the molecule type in the converter function
+        setattr(self.obj, self.sbol_term_suf, self.cell_val)
```

Comment:
this function now does the same as the basic function above so can just be commented out

Reply:
OK, will do.